.config/gitpod/Dockerfile

```diff
@@ -38,23 +36,37 @@ RUN PYTHON_VERSION=$(cut -d '=' -f 2 /tmp/python_version.config) && \
         vim \
         wget \
     && python3 -m pip install --no-cache-dir --upgrade pip \
-    && python3 -m pip install --no-cache-dir virtualenv \
-    # Install Docker
-    && curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
+    && python3 -m pip install --no-cache-dir virtualenv
+
+# Install Docker
+FROM builder as docker-installer
+
+RUN curl -fsSL https://download.docker.com/linux/ubuntu/gpg | gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg \
```

Comment:
DL4006 warning: Set the SHELL option -o pipefail before RUN with a pipe in it. If you are using /bin/sh in an alpine image or if your shell is symlinked to busybox then consider explicitly setting your SHELL to /bin/ash, or disable this check: For enhanced security, you can add `SHELL ["/bin/bash", "-o", "pipefail", "-c"]` before RUN commands that use pipes.

Reply:
@ThomasSanson 


I have just made the modification, including the SHELL I had to place before hadolint. Otherwise, the modification for DL3008 and DL3009 errors would not be applied anymore.

You can find the modifications in this commit:7db31daba6df5509a1e118598c24e828218b5c3e